src/printer.js

```diff
@@ -114,13 +114,13 @@ function genericPrint(path, options, printPath, args) {
   }
 
   if (needsParens) {
-    parts.unshift("( ");
+    parts.unshift("(", options.calypsoSpacing ? " " : "");
```

Comment:
just saw that you _almost_ do this later by creating `parenSpace`.  I'm still in favor of hoisting it one scope + of renaming to calypsoSpace to both line up with `calypsoSpacing` option and to jump out at maintainers of this package